components/public-api/go/config/config.go

```diff
@@ -26,5 +29,11 @@ type Configuration struct {
 	// Path to directory containing database configuration files
 	DatabaseConfigPath string `json:"databaseConfigPath"`
 
+	// RedisAddress configures the redis connection of this component
+	RedisAddress string `json:"redisAddress"`
```

Comment:
Let's make this a struct, which has an address as a property. I can imagine we may want to pass additional info (like creds) and having these sit under a single `redis` property makes it easier.

```
Redis struct {
	Address string `json:...`
}